config/config.go

```diff
@@ -42,3 +43,8 @@ func LoadConfig(file string) {
 		log.SetLevel(log.ErrorLevel)
 	}
 }
+
+// AbsolutePath returns the absolute file path under the specified data directory
+func AbsolutePath(name string) string {
+	return filepath.Join(viper.GetString("data_dir"), name)
```

Comment:
It would be better to add `filepath.Abs` before join. There would be an error from `Abs` but I think it is fine to throw it in `panic` since invalid path is not allowed in the program.